packages/react-components/src/templates/Layout.tsx

```diff
@@ -38,6 +38,8 @@ const headerMenuShownStyles = css({
 });
 
 const contentStyles = css({
+  backgroundColor: pearl.rgb,
```

Comment:
Please move below the positioning styles to inner styles. I'm not usually this pedantic, but this layout would be extremely hard to grasp without
![order](https://media4.giphy.com/media/kPCaLgYoCSxLiQ9qKY/giphy.gif)